File: 2335-minimum-amount-of-time-to-fill-cups/2335-minimum-amount-of-time-to-fill-cups.go
func fillCups(amount []int) int {
    res := 0
    for amount[0] > 0 || amount[1] > 0 || amount[2] > 0{
        sort.Sort(sort.Reverse(sort.IntSlice(amount)))
        res++
        amount[0]--
        amount[1]--
    }
    return res
}

File: 2335-minimum-amount-of-time-to-fill-cups/2335-minimum-amount-of-time-to-fill-cups.py
class Solution:
    def fillCups(self, amount) -> int:
        res = 0
        while amount[0] > 0 or amount[1] > 0 or amount[2] > 0:
            amount = sorted(amount, reverse=True)
            res += 1
            amount[0] -= 1
            amount[1] -= 1
        return res